### PR TITLE
grammar fixes and wording improvements

### PR DESCRIPTION
**Pull request by UnknownDevices**
Just improve some wordings and fix various grammatical errors in some Getting Started chapters.

### CODE REVIEW

File: src/fyrox/beginning/scene_and_scene_graph.md [hunk 1]
```diff
@@ -1,87 +1,79 @@
...
-state is backed by one or more animation playing or blending nodes. See the [respective chapter](../animation/absm_editor.md) 
+state is backed by one or more animation playing or blending nodes. See its [respective chapter](../animation/absm_editor.md) 
 for more info.
 
-Each of the node could be created either from the editor (`Create` on main menu) or programmatically via respective
-node builder (see [API docs](https://docs.rs/fyrox/latest/fyrox/scene/index.html) for more info). These scene nodes 
-allow you to build almost any kind of game. It is also possible to create your own types of nodes, but this is advanced 
-topic, and it will be covered in [next chapters](../scene/custom_node.md).
+Every node can be created either from the editor (through `Create` on the main menu, or through `Add Child` after right-clicking on 
+a game entitiy) or programmatically via their respective node builder (see [API docs](https://docs.rs/fyrox/latest/fyrox/scene/
```
Comment (mrDIMAS) [inline src/fyrox/beginning/scene_and_scene_graph.md:64]: typo `entitiy`

File: src/fyrox/beginning/scene_and_scene_graph.md [hunk 2]
```diff
@@ -1,87 +1,79 @@
...
-This node is intended to be used for **2D games** only.
-- [Sound](../sound/sound.md) - a sound source, it is universal for 2D and 3D. Spatial blend factor allows you to select
+This node is intended for use in **2D games** only.
+- [Sound](../sound/sound.md) - a sound source universal for 2D and 3D. Spatial blend factor allows you to select
 a proportion between 2D and 3D.
-- Listener - an audio receiver that captures the sound in a particular point in your scene and sends it to an audio
-context for processing and output to an audio playback device.
-- Animation Player - a container for multiple animations, it can play animations made in the 
+- Listener - an audio receiver that captures the sound at a particular point in your scene and sends it to an audio
+context for processing and outputing to an audio playback device.
```
Comment (mrDIMAS) [inline src/fyrox/beginning/scene_and_scene_graph.md:56]: typo `outputing`

**Review by mrDIMAS — CHANGES_REQUESTED** [review]
Thanks for the PR! There are two minor typos, could you please fix them and then I'll merge the PR. 

PS. Don't mind the failed tests, it is due to latest API changes in the engine.

File: src/fyrox/beginning/scene_and_scene_graph.md (hunk 2 above)
Comment (UnknownDevices) [reply src/fyrox/beginning/scene_and_scene_graph.md]: Fixed them, I must have forgotten run this file through a spell checker

**Pull request merged**